tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/LauncherApplicationTest.java

```diff
@@ -0,0 +1,206 @@
+import org.jboss.tools.common.launcher.reddeer.wizards.NewLauncherProjectWizardPage;
+import org.json.simple.JSONArray;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ *
+ * @author zcervink
+ * @see https://issues.jboss.org/browse/JBIDE-26520
+ *
+ */
+@RunWith(RedDeerSuite.class)
+public class LauncherApplicationTest {
+
+	private static final String CENTRAL_LABEL = "Red Hat Central";
+	private static final Logger log = Logger.getLogger(ExamplesOperator.class);
+	private static final String BLACKLIST_ERRORS_REGEXES_FILE = "resources/blacklist-launcher-application-errors-regexes.json";
+
+	@BeforeClass
+	public static void setup() {
+		new DefaultToolItem(new WorkbenchShell(), CENTRAL_LABEL).click();
+		// activate central editor
+		new DefaultEditor(CENTRAL_LABEL);
+	}
+
+	@After
+	public void teardown() {
+		WorkbenchShellHandler.getInstance().closeAllNonWorbenchShells();
+		for (Project p : new ProjectExplorer().getProjects()) {
+			p.delete(true);
+		}
+		new DefaultToolItem(new WorkbenchShell(), CENTRAL_LABEL).click();
+		// activate central editor
+		new DefaultEditor(CENTRAL_LABEL);
+	}
+
+	@Test
+	public void createNewLauncherApplicationProjectWithDefaultSettings() {
+		log.step("Import project start");
+		
+		NewLauncherProjectWizard wizard = new NewLauncherProjectWizard();
+		NewLauncherProjectWizardPage wizardPage = new NewLauncherProjectWizardPage(wizard);
+		
+		wizard.openWizardFromShellMenu();
+		wizardPage.setProjectName("project_1_default_settings");
+
+		wizard.finish(TimePeriod.getCustom(2500));
+		log.step("Import project finished");
+
+		checkForErrors();
+	}
+
+	@Test
+	public void createNewLauncherApplicationProjectWithCustomizedSettings() {
+		log.step("Import project start");
+
+		NewLauncherProjectWizard wizard = new NewLauncherProjectWizard();	
+		NewLauncherProjectWizardPage wizardPage = new NewLauncherProjectWizardPage(wizard);
+		
+		wizard.openWizardFromShellMenu();
+
+		wizardPage.setTargetMission("rest-http");
+		wizardPage.setTargetRuntime("vert.x redhat");
+		wizardPage.setProjectName("project_2_customized_settings");
+		wizardPage.toggleUseDefaultLocationCheckBox(true);
+		wizardPage.setCustomProjectLocation("/Users/zcervink/zc-custom-loc");
+		wizardPage.setArtifactId("my_customized_artifact_id");
+		wizardPage.setGroupId("my.customized.group.id");
+		wizardPage.setVersion("1.2.3-SNAPSHOT");
+
+		wizard.finish(TimePeriod.getCustom(2500));
+		log.step("Import project finished");
+
+		checkForErrors();
+	}
+
+	@Test
+	public void testFinishButtonDisabledInLauncherApplicationProjectWizard() {
+		log.step("Import project start");
+
+		NewLauncherProjectWizard wizard = new NewLauncherProjectWizard();	
+		NewLauncherProjectWizardPage wizardPage = new NewLauncherProjectWizardPage(wizard);
+		wizard.openWizardFromShellMenu();
+
+		// PROJECT NAME not filled test
+		assertFalse(wizardPage.isFinishButtonEnabled());
```

Comment:
Also, please move isFinishButtonEnabled from page into wizard dialog class same as impl. of finish methods. Thanks.